Azure/azure-cli
# [Compute] az vm/vmss create: Add support for specialized image version

**Description**  

Add `--specialized` parameter for `az vm create` and `az vmss create` to support creating vm/vmss from a specialized image version

**Testing Guide**  

Create a VM from a specialized image version.
```
az vm create -g MyResourceGroup -n MyVm --image $id --specialized
```
Create a Linux VM scale set from a specialized image version.
```
az vmss create -n MyVmss -g MyResourceGroup --image $id --specialized
```
See `test_specialized_image()` for more detail about how to prepare specialized image version.

**History Notes**  


[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
